piexif/_load.py

```diff
@@ -103,35 +103,74 @@ def __init__(self, data):
                 else:
                     raise InvalidImageDataError("Given file is neither JPEG nor TIFF.")
 
+    def _unpack_from(self, format, pointer):
+        return unpack_from(self.endian_mark + format, self.tiftag, pointer)
+
+    def _read_tag(self, pointer):
+        tag, value_type, value_num = self._unpack_from("HHL", pointer)
+        # Treat unknown types as `Undefined`
+        value_length = TYPE_LENGTH.get(value_type, 1) * value_num
+        if value_length > 4:
+            data_pointer = self._unpack_from("L", pointer + 8)[0]
```

Comment:
Выше ещё два места в `load` и ниже tag_count.

```suggestion
            data_pointer, = self._unpack_from("L", pointer + 8)
```